opentracing-util/src/main/java/io/opentracing/util/ThreadLocalActiveSpan.java

```diff
@@ -22,13 +22,13 @@
 import java.util.Map;
 import java.util.concurrent.atomic.AtomicInteger;
 
- /**
-  * {@link ThreadLocalActiveSpan} is a simple {@link ActiveSpan} implementation that relies on Java's
-  * thread-local storage primitive.
-  *
-  * @see ActiveSpanSource
-  * @see Tracer#activeSpan()
-  */
+/**
```

Comment:
Nope, must have been an IntelliJ autoformat.
By the way, I agree with you on the license-only commit; that was my intention, the formatting 'came free' by the tooling :wink: